x/claims/keeper/invariants.go

```diff
@@ -28,7 +28,7 @@ func (k Keeper) ClaimsInvariant() sdk.Invariant {
 		// iterate over all the claim records and sum the unclaimed amounts
 		k.IterateClaimsRecords(ctx, func(_ sdk.AccAddress, cr types.ClaimsRecord) bool {
 			// IMPORTANT: use Dec to prevent truncation errors
-			initialClaimablePerAction := cr.InitialClaimableAmount.ToDec().Quo(numActions)
+			initialClaimablePerAction := sdk.NewDecFromInt(cr.InitialClaimableAmount).Quo(numActions)
```

Comment:
why did this change?

Reply:
There is no ToDec(), so, we have to resort to NewDecFromInt(), it is the same anyway.